var/spack/repos/builtin/packages/py-aiosqlite/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyAiosqlite(PythonPackage):
+    """asyncio bridge to the standard sqlite3 module"""
+
+    homepage = "https://aiosqlite.omnilib.dev"
+    pypi     = "aiosqlite/aiosqlite-0.17.0.tar.gz"
+
+    version('0.17.0', sha256='f0e6acc24bc4864149267ac82fb46dfb3be4455f99fe21df82609cc6e6baee51')
+
+    depends_on('python@3.6:', type=('build', 'run'))
```

Comment:
Needs:
```python
depends_on('py-flit-core@2:3', type='build')
```
This isn't used at the moment because we run `python setup.py build|install`, but once #27798 is merged we'll be using `pip install` and `pip` reads from the `pyproject.toml`.